qa/suites/krbd/basic/tasks/krbd_default_map.yaml

```
@@ -0,0 +1,5 @@
+tasks:
+- cram:
+    clients:
+      client.0:
+      - qa/rbd/krbd_default_map.t
```

```suggestion
      - qa/rbd/krbd_default_map_options.t
```
and the same for the yaml filename.

---

Done.